docs/ops/movement/StridedSlice_1.md

```diff
@@ -51,19 +51,19 @@
 
 **Inputs**:
 
-*   **1**: Multidimensional input tensor to be sliced. Required.
+*   **1**: input tensor to be sliced of type `T` and arbitrary shape. **Required.**
 
-*   **2**: `begin` input - 1D input tensor with begin indexes for input tensor slicing. Required.
-           Out-of-bounds values are silently clamped. If `begin_mask[i]` is 1, the value of `begin[i]` is ignored
-           and the range of the appropriate dimension starts from 0.
-           Negative values mean indexing starts from the end. For example, if `foo=[1,2,3]`, `begin[0]=-1` means `begin[0]=3`.
+*   **2**: `begin` - 1D tensor of type `T_IND` with begin indexes for input tensor slicing. **Required.**  
+    Out-of-bounds values are silently clamped. If `begin_mask[i]` is `1`, the value of `begin[i]` is ignored and the range of the appropriate dimension starts from `0`. Negative values mean indexing starts from the end. For example, if `foo=[1,2,3]`, `begin[0]=-1` means `begin[0]=3`.
 
-*   **3**: `end` input - 1D input tensor with end indexes for input tensor slicing. Required.
-           Out-of-bounds values will be silently clamped. If `end_mask[i]` is 1, the value of `end[i]` is ignored
-           and the full range of the appropriate dimension is used instead.
-           Negative values mean indexing starts from the end. For example, if `foo=[1,2,3]`, `end[0]=-1` means `end[0]=3`.
+*   **3**: `end` - 1D tensor of type `T_IND` with end indexes for input tensor slicing. **Required.**  
+    Out-of-bounds values will be silently clamped. If `end_mask[i]` is `1`, the value of `end[i]` is ignored and the full range of the appropriate dimension is used instead. Negative values mean indexing starts from the end. For example, if `foo=[1,2,3]`, `end[0]=-1` means `end[0]=3`.
 
-*   **4**: `stride` input - 1D input tensor with strides. Optional.
+*   **4**: `stride` - 1D tensor of type `T_IND` with strides. **Optional.**
+
+**Types**
+* *T*: any supported floating point type.
+* *T_IND*: any supported integer type.
 
 **Example**
 ```xml
```

Comment:
Can we add more examples with some complex cases with `new_axis_mask` and `begin/end_mask`?

Reply:
Added new examples in this [commit](https://github.com/openvinotoolkit/openvino/pull/5887/commits/8f8cca0722c9953e6b46d45fc9b9794c5e5f0c21).